aspnet/mvc/views/forms/sample/final/Controllers/HomeController.cs

```diff
@@ -0,0 +1,120 @@
+﻿using FormsTagHelper.ViewModels;
+using Microsoft.AspNet.Mvc;
+
+namespace FormsTH
```

Comment:
Be consistent about the namespaces throughout this sample e.g. this should be in the `FormsTagHelper.Controllers` namespace, alongside `DemoController`.